pyproject.toml

```diff
@@ -34,6 +34,7 @@ dependencies = [
   "accelerate==0.34.2",
   "hf-transfer==0.1.8",
   "cachetools~=5.5",
+  "strenum; python_version<'3.11'",
```

Comment:
Can this be pinned to `strenum==0.4.15` ?

Reply:
Done